src/python/pants/backend/python/goals/export.py

```diff
@@ -77,74 +81,128 @@ def debug_hint(self) -> str | None:
         return self.resolve_name
 
 
+class ExportPluginOptions:
+    symlink_python_virtualenv = BoolOption(
+        default=False,
```

Comment:
We did change the implementation from symlinking to pipifying without deprecation... but going back probably does require deprecation since people may be assuming they can run pip in the resulting venv?

Reply:
> people may be assuming they can run pip in the resulting venv?

I can assure you they do expect it to work, whether they know that or not. I've seen more than enough `pip install`-slingers to make a grown man cry.